reactioncommerce/reaction
# Bugfix: low quantity/sold out flags not saving correctly in db

Resolves #4328 
Impact: minor
Type: bugfix

## Issue
When a Product is published to the Catalog collection the nested variants and options will always have inventory status properties of false.

## Solution
Update/refactor `xformVariant` to use `isSoldOut` and `isLowQuantity` util functions when setting properties

## Testing
1. Set option/variant quantity to 0 and publish
2. Verify corresponding property is set correctly in `Catalog` collection
3. Repeat process for low quantity property.
